index.html

```diff
@@ -6,6 +6,8 @@
 	<script src="js/ShowAndNavigate.js" defer></script>
```

Comment:
remove 'defer' attribute because defer's definition varies spec to spec, and even in the most recent specs doesn't offer a useful guarantee